src/systems/particles.rs

```diff
@@ -0,0 +1,53 @@
+use crate::{Particle, ParticleEmitter, Velocity};
+use bevy::prelude::*;
+use probability::distribution::Gaussian;
+use probability::prelude::*;
+use rand::distributions::{Bernoulli, Distribution};
+use std::time::Duration;
+
+/// emit new particles from entites with the `ParticleEmitter` component
+pub fn particle_emission_system(
+    mut commands: Commands,
+    query: Query<(&ParticleEmitter, &Transform, &Sprite, &Velocity)>,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+) {
+    for (_particle_emmiter, transform, sprite, velocity) in query.iter() {
+        let d = Bernoulli::new(0.3).unwrap();
+        if d.sample(&mut rand::thread_rng()) {
+            let mut source = source::default();
+            let random_direction_dist = Gaussian::new(0.0, 1.0);
+            let sampler = Independent(&random_direction_dist, &mut source);
+            let samples = sampler.take(2).collect::<Vec<f64>>();
+            commands
+                .spawn_bundle(SpriteBundle {
+                    material: materials.add(Color::rgba(1.0, 1.0, 1.0, 1.0).into()),
```

Comment:
@jeremyadamsfisher can we reduce the opacity here so I can actually see the ball :P 
Love this PR